### PR TITLE
add per capita to url

### PR DESCRIPTION
**Pull request by qlj-lijuan**

### CODE REVIEW

File: static/js/tools/timeline_util.ts [hunk 1]
```diff
@@ -200,40 +205,50 @@ function getStatsVarInfo(dcids: string[]) {
...
-const promises = [];
-// ToDo: read the set of statsvars available for multiple dcids from server side
-for (const dcid of dcids) {
-  promises.push(
-    axios.get("/api/place/statsvars/" + dcid).then((resp) => {
-      return resp.data;
-    })
-  );
+function getPC() {
+  const vars = getUrlVars() as VarUrl;;
```
Comment (shifucun) [inline static/js/tools/timeline_util.ts:209]: extra ;

File: static/js/tools/timeline_util.ts [hunk 2]
```diff
@@ -200,40 +205,50 @@ function getStatsVarInfo(dcids: string[]) {
...
+      if (statvars.has(MAPPING[key])) {
+        statvars.add(key);
+      }
+    })
+    return statvars;
+  }) as Promise<Set<string>>;
 }
 
 export {
   updateUrlStatsVar,
```
Comment (shifucun) [inline static/js/tools/timeline_util.ts:244]: at this stage, maybe should combine these 3 update function into one function and have an interface/object to wrap <place, statsvar, pc>

File: static/js/tools/timeline_util.ts [hunk 3]
```diff
@@ -200,40 +205,50 @@ function getStatsVarInfo(dcids: string[]) {
...
   updateUrlPlace,
+  updateUrlPC,
   parseStatVarPath,
   parsePlace,
   getStatsVarInfo,
   getPlaceNames,
+  getPC,
```
Comment (shifucun) [inline static/js/tools/timeline_util.ts:251]: same to combine 3 get functions into one

File: static/js/tools/timeline_util.ts (hunk 1 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_util.ts]: removed

File: static/js/tools/timeline_util.ts (hunk 2 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_util.ts]: done.

File: static/js/tools/timeline_util.ts (hunk 3 above)
Comment (qlj-lijuan) [reply static/js/tools/timeline_util.ts]: done.

**Pull request merged**